Firestore/core/src/model/mutable_document.cc

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/model/mutable_document.h"
+
+namespace firebase {
+namespace firestore {
+namespace model {
+
+/* static */
+MutableDocument MutableDocument::InvalidDocument(
+    const firebase::firestore::model::DocumentKey& document_key) {
```

Comment:
I think you can remove all of the namespace prefixes here, they are from the same `model`.

Reply:
Fixed. Sorry for not catching this myself.